app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -161,30 +161,34 @@ class QuickJump extends ConsumerWidget {
       body.add(QuickActionsBuilder(navigateTo: navigateTo));
     }
 
-    return Padding(
-      padding: const EdgeInsets.all(16.0),
-      child: Column(
-        mainAxisSize: MainAxisSize.min,
-        mainAxisAlignment: MainAxisAlignment.start,
-        children: <Widget>[
-          TextField(
-            autofocus: true,
-            decoration: const InputDecoration(
-              border: InputBorder.none,
-              focusedBorder: UnderlineInputBorder(),
-              prefixIcon: Icon(
-                Atlas.magnifying_glass_thin,
-                color: Colors.white,
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: <Widget>[
+            Expanded(
+              child: TextField(
```

Comment:
nope. this is wrong. for pointer-devices, we do want the content close together. Only on the phone where the button is on the bottom do we want the quick-actions to be at the bottom.

This on desktop, is wrong:
![grafik](https://github.com/acterglobal/a3/assets/40496/e60e2dc4-4113-4e7d-9220-dab2298285cb)

Please revert.

Reply:
Actually this was misplaced here. I was looking this view on Android and it doesn't appear well .i.e. search field is overflowing the system status bars and obscuring the view. This should be refactored, but on the other hand, will remove ```Expanded```.